lib/moebius/runner.ex

```diff
@@ -1,34 +1,51 @@
 defmodule Moebius.Runner do
+  use GenServer
+
+  def start_link do
+    GenServer.start_link(__MODULE__, open_connection, name: Moby)
+  end
+
   @moduledoc """
   The main execution bits are in here.
   """
 
-  @doc """
-  Spawn a Postgrex worker to run our query using the config specified in /config
-  """
-  def connect do
+  defp open_connection do
     extensions = [{Postgrex.Extensions.JSON, library: Poison}]
 
-    Application.get_env(:moebius, :connection)
-      |> Keyword.update(:extensions, extensions, &(&1 ++ extensions))
-      |> Postgrex.Connection.start_link
+    {:ok, pid} = Application.get_env(:moebius, :connection)
+              |> Keyword.update(:extensions, extensions, &(&1 ++ extensions))
+              |> Postgrex.Connection.start_link
+
+    %{pid: pid}
```

Comment:
This should be done in the `init` callback. Otherwise the connection will be linked to the supervisor which is not what you want.